commands/command_track.go

```diff
@@ -19,6 +19,8 @@ var (
 	}
 )
 
+// trackCommand takes a list of paths as an argument, and adds each path to the default
+// attribtues file (.gitattributes), if it's not already exists.
```

Comment:
attribtues → attributes

Also, not completely true. It sounds like there is only one `.gitattributes` file.  Actually there is one per directory.